src/pages/Index.vue

```diff
@@ -2,18 +2,115 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    <g-image alt="Example image" src="~/favicon.png" width="135" />
+    <header class="header">
```

Comment:
The header is already reserved for the navbar. I would not use a header there

Reply:
Okay

> Can you see https://app.netlify.com/sites/hungry-aryabhata-193bcf/deploys/6203acfeeca84700074261cb ?

2:05:12 PM: [ERROR] Deploy logs are currently unavailable. We are working on resolving the issue.

That's the response I see there